### ISSUE TITLE
iTerm2 verision 

### ISSUE THREAD

**Issue by enzime4u** [mao-santaella-rs/NightWolfTheme#25]
I tottally love your theme with the dark-blue , can I please ask for a iTerm verision ? Thank you

**Comment by mao-santaella-rs** [comment]
Hi, @enzime4u do you mean the background color of iTerm? thank you for using the theme.

**Comment by enzime4u** [comment]
I mean , if we can make an iTerm2 version of the Night Wolf !!! Your team is my day-to-day use , and belive me I tried and tweaked 100+ :))

**Comment by mao-santaella-rs** [comment]
I don't think I'm following, I think you mean by "we" that you want to make the iTerm version of the theme with your team or something if this is the case, sure make the pull request and I'll take a look; or if you mean by "we" that you want that between you and I make the iTerm version of the theme, sure we just need the color schema of iTerm and try to make a version of it.
Also, I don't understand if you just want either a version with just the background color of iTerm or with the text colors too.
let me know, thanks again.

**Comment by enzime4u** [comment]
Sorry, for the misunderstandings ! by WE i was talking if YOU and ME, can try to make a iTerm version. 
I was thinking about a whole version for iTerm, with syntax and colors !

**Comment by mao-santaella-rs** [comment]
Absolutely, however, I think we can make a new theme called ITerm Theme, what do you think? and we can take the build process from the night wolf and make the new theme: Also is there a chance to know all the color schema from iTerm?

**Comment by enzime4u** [comment]
Amazing !!! Need to dig into that ..... will google a little

**Comment by enzime4u** [comment]
`<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>Ansi 0 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.15487705171108246</real>
		<key>Green Component</key>
		<real>0.11203418672084808</real>
		<key>Red Component</key>
		<real>0.12318958342075348</real>
	</dict>
	<key>Ansi 1 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.24190764129161835</real>
		<key>Green Component</key>
		<real>0.22509026527404785</real>
		<key>Red Component</key>
		<real>0.84957921504974365</real>
	</dict>
	<key>Ansi 10 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.4497399628162384</real>
		<key>Green Component</key>
		<real>0.80302643775939941</real>
		<key>Red Component</key>
		<real>0.17663580179214478</real>
	</dict>
	<key>Ansi 11 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.43218505382537842</real>
		<key>Green Component</key>
		<real>0.71836566925048828</real>
		<key>Red Component</key>
		<real>0.85048472881317139</real>
	</dict>
	<key>Ansi 12 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.51579737663269043</real>
		<key>Green Component</key>
		<real>0.75985312461853027</real>
		<key>Red Component</key>
		<real>1</real>
	</dict>
	<key>Ansi 13 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.25254109501838684</real>
		<key>Green Component</key>
		<real>0.5535169243812561</real>
		<key>Red Component</key>
		<real>0.86943024396896362</real>
	</dict>
	<key>Ansi 14 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>1</real>
		<key>Green Component</key>
		<real>0.53426611423492432</real>
		<key>Red Component</key>
		<real>0.14025618135929108</real>
	</dict>
	<key>Ansi 15 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>1</real>
		<key>Green Component</key>
		<real>0.89618134498596191</real>
		<key>Red Component</key>
		<real>0.91595911979675293</real>
	</dict>
	<key>Ansi 2 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.4497399628162384</real>
		<key>Green Component</key>
		<real>0.80302643775939941</real>
		<key>Red Component</key>
		<real>0.17663580179214478</real>
	</dict>
	<key>Ansi 3 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.43218505382537842</real>
		<key>Green Component</key>
		<real>0.71836566925048828</real>
		<key>Red Component</key>
		<real>0.85048472881317139</real>
	</dict>
	<key>Ansi 4 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.51579737663269043</real>
		<key>Green Component</key>
		<real>0.75985312461853027</real>
		<key>Red Component</key>
		<real>1</real>
	</dict>
	<key>Ansi 5 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.25254109501838684</real>
		<key>Green Component</key>
		<real>0.5535169243812561</real>
		<key>Red Component</key>
		<real>0.86943024396896362</real>
	</dict>
	<key>Ansi 6 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>1</real>
		<key>Green Component</key>
		<real>0.53426611423492432</real>
		<key>Red Component</key>
		<real>0.14025618135929108</real>
	</dict>
	<key>Ansi 7 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>1</real>
		<key>Green Component</key>
		<real>0.63088703155517578</real>
		<key>Red Component</key>
		<real>0.71658247709274292</real>
	</dict>
	<key>Ansi 8 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.27691897749900818</real>
		<key>Green Component</key>
		<real>0.19346600770950317</real>
		<key>Red Component</key>
		<real>0.20805245637893677</real>
	</dict>
	<key>Ansi 9 Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.24190764129161835</real>
		<key>Green Component</key>
		<real>0.22509026527404785</real>
		<key>Red Component</key>
		<real>0.84957921504974365</real>
	</dict>
	<key>Background Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.15487705171108246</real>
		<key>Green Component</key>
		<real>0.11203418672084808</real>
		<key>Red Component</key>
		<real>0.12318958342075348</real>
	</dict>
	<key>Bold Color</key>
	<dict>
		<key>Blue Component</key>
		<real>1</real>
		<key>Green Component</key>
		<real>0.63585507869720459</real>
		<key>Red Component</key>
		<real>0.71660345792770386</real>
	</dict>
	<key>Cursor Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.222110316157341</real>
		<key>Green Component</key>
		<real>0.59581488370895386</real>
		<key>Red Component</key>
		<real>1</real>
	</dict>
	<key>Cursor Text Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.15487705171108246</real>
		<key>Green Component</key>
		<real>0.11203418672084808</real>
		<key>Red Component</key>
		<real>0.12318958342075348</real>
	</dict>
	<key>Foreground Color</key>
	<dict>
		<key>Blue Component</key>
		<real>1</real>
		<key>Green Component</key>
		<real>0.63088703155517578</real>
		<key>Red Component</key>
		<real>0.71658247709274292</real>
	</dict>
	<key>Selected Text Color</key>
	<dict>
		<key>Blue Component</key>
		<real>1</real>
		<key>Green Component</key>
		<real>0.63585507869720459</real>
		<key>Red Component</key>
		<real>0.71660345792770386</real>
	</dict>
	<key>Selection Color</key>
	<dict>
		<key>Blue Component</key>
		<real>0.27691897749900818</real>
		<key>Green Component</key>
		<real>0.19346600770950317</real>
		<key>Red Component</key>
		<real>0.20805245637893677</real>
	</dict>
</dict>
</plist>
`
